apps/webapp/src/state/staking.ts

```diff
@@ -4,16 +4,28 @@ import { getDelegationsForAccount } from '../fetchers/staking';
 import {
   VotingPowerAsIntegerPercentage,
   getAmount,
+  getDisplayDenomExponent,
   getDisplayDenomFromView,
+  getRateData,
   getValidatorInfoFromValueView,
   getVotingPowerByValidatorInfo,
   getVotingPowerFromValidatorInfo,
   joinLoHiAmount,
+  toBaseUnit,
 } from '@penumbra-zone/types';
 import { ValueView } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/asset/v1/asset_pb';
 import { getBalancesByAccount } from '../fetchers/balances/by-account';
-import { STAKING_TOKEN } from '@penumbra-zone/constants';
+import { STAKING_TOKEN, localAssets } from '@penumbra-zone/constants';
 import { AddressIndex } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/keys/v1/keys_pb';
+import { TransactionToast } from '@penumbra-zone/ui';
+import { authWitnessBuild, broadcast, getTxHash, plan, userDeniedTransaction } from './helpers';
+import { TransactionPlannerRequest } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/view/v1/view_pb';
+import { BigNumber } from 'bignumber.js';
+
+const STAKING_TOKEN_DISPLAY_DENOM_EXPONENT = (() => {
+  const stakingAsset = localAssets.find(asset => asset.display === STAKING_TOKEN);
```

Comment:
Not important for this PR, but I have a hunch we probably can't rely upon a direct `localAssets` import and need to query the extension for a list of assets it knows of. Need to have a thought about this. Don't think we have an RPC method to give us "all metadata you know of" (aka the `ASSETS` table).

Reply:
Correction, we have `AssetsRequest` we could use. Think this is worthwhile to use here instead of the hardcoded registry.

Reply:
do we actually need to query this way? there's only one staking token iirc and its id should be static, we will either have the md or could query by id

Reply:
@grod220 I was thinking along the same lines as @turbocrime — that there's only one staking token, so no need to make a query. i.e., this can be hardcoded. Thoughts?

Reply:
Going to leave this as-is for now so I can merge, but let's revisit Monday if I'm missing something .